pkg/agent/agent.go

```diff
@@ -208,7 +218,47 @@ func (a *Agent) attest(ctx context.Context, sto storage.Storage, cat catalog.Cat
 		ServerAddress:     a.c.ServerAddress,
 		NodeAttestor:      na,
 	}
-	return node_attestor.New(&config).Attest(ctx)
+
```

Comment:
I would prefer to have the retry logic in the caller, and not inside this function. We want to retry this when called from a specific place, rather than always.